### DISCUSSION TITLE
@import error in child-theme.scss

### DISCUSSION THREAD

**Discussion by gurds91** [holger1411/understrap#1801] [Q&A] [answered]
Hi guys,

tried to import a style sheet into main chid-theme.scss using @import but import turned yellow and squiggly and error ;(

it says media query expected when hovering over it, but I don't know why they expect a media query. Not sure how to solve, have attached issue 
![import error scss](https://user-images.githubusercontent.com/107938466/176792972-a944c221-926c-429f-a9ff-ffa50808a4c6.jpg)

**Comment by IanDelMar** [comment, marked as answer]
The semicolon after `@import "theme/certification-temp"` is missing.

**Reply by gurds91** [reply]
thanks a lot ;) Didn't spot it

**Reply by IanDelMar** [reply]
You're welcome. Consider to mark the topic as being answered so others know that you no longer need support concerning this issue.
